From 51170b52d15256d4aaf74ed6eea9a9297f5d595c Mon Sep 17 00:00:00 2001
From: Helmut Grohne <helmut@subdivi.de>
Date: Wed, 23 Apr 2025 09:54:28 +0100
Subject: [PATCH] autotools: install shell completion files on cross build

 Before 8.13.0, it was not possible to generate them as it required
 calling the compiled binary, but this has been fixed.

Co-authored-by: Samuel Henrique <samueloph@debian.org>

Closes #17159
---
 scripts/Makefile.am | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/scripts/Makefile.am b/scripts/Makefile.am
index 705a77a09e75..8cd5b77b78e5 100644
--- a/scripts/Makefile.am
+++ b/scripts/Makefile.am
@@ -57,9 +57,6 @@ $(FISH_COMPLETION_FUNCTION_FILENAME): completion.pl
 endif
 
 install-data-local:
-if CROSSCOMPILING
-	@echo 'NOTICE: we cannot install completion scripts when cross-compiling'
-else # if not cross-compiling:
 if USE_ZSH_COMPLETION
 	if test -n "$(PERL)"; then \
 	  $(MKDIR_P) $(DESTDIR)$(ZSH_FUNCTIONS_DIR); \
@@ -72,7 +69,6 @@ if USE_FISH_COMPLETION
 	  $(INSTALL_DATA) $(FISH_COMPLETION_FUNCTION_FILENAME) $(DESTDIR)$(FISH_FUNCTIONS_DIR)/$(FISH_COMPLETION_FUNCTION_FILENAME); \
 	fi
 endif
-endif
 
 distclean:
 	rm -f $(CLEANFILES)
